pyasic/miners/backends/braiins_os.py

```diff
@@ -720,6 +720,10 @@ async def upgrade_firmware(self, file: Path):
             "_get_uptime",
             [RPCAPICommand("rpc_summary", "summary")],
         ),
+        str(DataOptions.POOLS): DataFunction(
```

Comment:
Can you remove the get_pools implementation for bosminer from this?  Want to keep this PR atomic as much as possible.  Any other get_pools implementatikons already done should also be updated to use the new PoolUrl dataclass.

Reply:
I have rebased and dropped the unwanted commits.